underscore.js

```diff
@@ -1213,6 +1213,11 @@
     // Assume equality for cyclic structures. The algorithm for detecting cyclic
     // structures is adapted from ES 5.1 section 15.12.3, abstract operation `JO`.
 
+    // Compare array lengths to determine if a deep comparison is necessary.
+    if (areArrays && a.length !== b.length) {
+      return false;
+    }
+    
```

Comment:
The problem seems to be "from unknown repository" at the top of the PR. Guess I need to resubmit the PR, as that's not something I can change... Sigh.